src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier supplier = new ColorSupplier();
+        Color randomColor = supplier.getRandomColor();
+        Random rnd = new Random();
+        int sum = rnd.nextInt(100);
+        ball.setColor(randomColor);
+        ball.setNumber(sum);
+        return ball;
```

Comment:
Create variables for new Random() and new ColorSuplier() as I showed in the comment above. Also make a constant for 100